docs/schemas

```diff
@@ -0,0 +1 @@
+../src/pyhf/schemas
```

Comment:
what's this? looks like you made this a symlink, and you shouldn't. symlinks don't work with TCP/IP.

Reply:
I was copying what you did in PR #452 https://github.com/diana-hep/pyhf/pull/452/files#diff-03602e9b79dc3f2cb4d9905f64179dbb. Why did you add it there if it isn't needed?

Reply:
Oh, hrmm... Maybe this PR looks awkward. Because it's shown as a _new file_ created.

Reply:
> Maybe this PR looks awkward. Because it's shown as a _new file_ created.

I agree. I was very confused by this behavior as well, but when you [view the file history it shows your previous versions](https://github.com/diana-hep/pyhf/commits/3d16b22be1f6f0a0e5f5d14271476a004ce5d7f0/docs/schemas), even though it is being shown as being "created".

Reply:
Most likely because of how GH treats this kind of file.